python/tests/test_logistic_regression.py

```diff
@@ -2009,7 +2011,9 @@ def sparse_to_df(X: csr_matrix, y: List[float]) -> DataFrame:
         gpu_lr = LogisticRegression(float32_inputs=float32_inputs, **est_params)
         cpu_lr = SparkLogisticRegression(**est_params)
 
+        gpu_lr.cuml_params["_convert_index"] = _convert_index
```

Comment:
Would be good to leave a comment about this param setting.

Reply:
Yeah, done. please check.